test/fixtures/no_header.po

```diff
@@ -1,8 +1,8 @@
 # some comment
 
-msgid "First id, no header"
-msgstr ""
+msgid   "First id, no header"
+msgstr  ""
```

Comment:
Please don't modify existing test fixtures: we need those to verify that the old cases still work.
Leave these untouched and add new ones + a new test case.

Apart from that this looks perfect, so with those changes I'll quickly merge and roll a release.

Reply:
Done